pkg/backup/item_backupper.go

```diff
@@ -384,6 +385,7 @@ const (
 	awsEbsCsiZoneKey = "topology.ebs.csi.aws.com/zone"
 	azureCsiZoneKey  = "topology.disk.csi.azure.com/zone"
 	gkeCsiZoneKey    = "topology.gke.io/zone"
+	zoneSeparator    = "__"
```

Comment:
is the zone separator for gcp only or all cloud providers?

Reply:
GKE only. I will update